packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -323,6 +328,42 @@ function errorMessageFromResponse(
     : '';
 }
 
+function retryDestination(
+  destinationName: string
+): Middleware<AxiosResponse<DestinationJson>, HttpMiddlewareContext> {
+  return options => () => {
+    let retryCount = 1;
+    return asyncRetry.default(
```

Comment:
[q] Is there any waiting time between retries? I would assume ~100ms between request could be helpful...

Reply:
There is a 1000ms wait before the first retry (default). Thereafter, it uses an exponential backoff strategy. As explained [here](https://github.com/tim-kos/node-retry#tutorial): With the default settings (10 in async-retry), this means the last attempt is made after 17 minutes and 3 seconds.

Reply:
That seems pretty long... maybe factor 3 and starting with 100ms would be better? Does not need to be changed in this PR though.

Reply:
I agree, I also had the same thought working on tests combining retry and cb. Will discuss with @FrankEssenberger since we also don't expose other options to the user.

Reply:
I thought we do 3 retries, let's discuss why it takes so long.

Reply:
Yes the default is 3 in SDK. The explanation in my first comment was about the default behavior in async-retry (where retry count is 10 and it takes 17 mins or so to finish). Even in the SDK, it's possible to set a higher value. I wanted to test a scenario where > 10 retries opens cb for 5xx error, and the test did take around 17 mins to finish.

Reply:
As discussed directly we will keep the options simple first. If there is feedback that the retry should have more options we will most likely add the union type `number | fullOptions` so that users can provide both.